src/library/FOSSBilling/CentralAlerts.php

```diff
@@ -46,7 +35,9 @@ public function getDi(): ?Container
      */
     public function getAlerts(): array
     {
-        $alerts = $this->di['cache']->get('CentralAlerts.getAlerts', function (ItemInterface $item) {
```

Comment:
I'm actually in favor of keeping this one..
It's pretty minor, but I like the peace of mind knowing we only have one potential spot to update if we decide on changing something about the cache implementation. (different path, settings, adapter, whatever)

And again it's very minor, but cron prunes the cache and if we accidentally end up with two different definitions for the cache implementation it will only correctly prune the one that's set in the DI

Reply:
That's a fair point - I'll revert this change and leave those using `cache` via the DI. 🙂